compiler/rustc_codegen_llvm/src/llvm/enzyme_ffi.rs

```diff
@@ -0,0 +1,39 @@
+#![allow(non_camel_case_types)]
+
+use libc::{c_char, c_uint, size_t};
+
+use super::ffi::{Attribute, BasicBlock, Builder, Metadata, Module, Type, Value};
+extern "C" {
+    // Enzyme
+    pub fn LLVMRustHasMetadata(I: &Value, KindID: c_uint) -> bool;
+    pub fn LLVMRustEraseInstBefore(BB: &BasicBlock, I: &Value);
+    pub fn LLVMRustGetLastInstruction<'a>(BB: &BasicBlock) -> Option<&'a Value>;
+    pub fn LLVMRustDIGetInstMetadata(I: &Value) -> &Metadata;
+    pub fn LLVMRustEraseInstFromParent(V: &Value);
+    pub fn LLVMRustGetTerminator<'a>(B: &BasicBlock) -> &'a Value;
+    pub fn LLVMGetFunctionCallConv(F: &Value) -> c_uint;
+
+    pub fn LLVMGetReturnType(T: &Type) -> &Type;
+    pub fn LLVMDumpModule(M: &Module);
+    pub fn LLVMCountStructElementTypes(T: &Type) -> c_uint;
+    pub fn LLVMVerifyFunction(V: &Value, action: LLVMVerifierFailureAction) -> bool;
+    pub fn LLVMGetParams(Fnc: &Value, parms: *mut &Value);
+    pub fn LLVMBuildCall2<'a>(
+        arg1: &Builder<'a>,
+        ty: &Type,
+        func: &Value,
+        args: *const &Value,
+        num_args: size_t,
```

Comment:
this is unsigned, not size_t

Reply:
`c_uint`, not `usize`. But you removed usage in recent commit, so entire extern `LLVMBuildCall2` can be removed?

Reply:
`LLVMIsEnumAttribute`,`LLVMIsStringAttribute`, `LLVMCountStructElementTypes`,`LLVMDumpModule` also looks unused. Or they placed for future usage?